docs/docs/reference/release-notes/v4.6/index.md

```diff
@@ -0,0 +1,51 @@
+---
+date: "2022-01-25"
+version: "4.6.0"
+title: "v4.6 Release Notes"
+---
+
+Welcome to `gatsby@4.6.0` release (January 2022 #2)
+
+Key highlights of this release:
+
+- [Performance update: Reuse cache across instances](#reuse-cache-across-instances).
+- [Improve live editing experience for images in markdown files](#improve-live-editing-experience-for-image-in-markdown-files)
+
+## Reuse cache across instances
+
+Walking nodes to track inline objects is very expensive. Caching this across instances of the graphql query sped up subsequent queries by ~10-15% (depending on complexity of nodes). Every performance improvement is important to use as we constantly look for way to make Gatsby faster than ever 🚀
```

Comment:
The first two sentences will most people probably not understand as they don't know what nodes, inline objects etc. are. The last sentence can be deleted, we don't have to type this out.

Generally speaking you don't just want to copy the description from the PR but frame it in another way so that all users can directly get it.

That's why we have the internal doc with this quote:

```
Good questions for Release Notes:

- Who is the target audience?
- What changed in this release? Why?
- How does the change impact the user?
- Is the tl;dr clear?

tl;dr: "Loads faster with fewer crashes"
Detail: "Added list virtualization. Reduced OOMs by 20%" etc.
```

If you're not sure how to exactly write it, you'll have to reach out to the author of the PR (in this case @KyleAMathews) to help you write it.